MeesGame/Gameplay/GameObjects/TileField.cs

```diff
@@ -135,15 +135,41 @@ public void UpdateGraphics()
 
         public void RevealArea(Point a)
         {
+            revealAroundTile(a);
```

Comment:
Limit the revealPath function to some amount of tiles and also reveal tiles in non-orthagonal directions (prisons may have open areas).